src/Post/postsController.js

```diff
@@ -1,10 +1,21 @@
+
 import postsService from "./postsService.js"
 
 class PostsController {
+    async getAll(req, res) {
```

Comment:
use then().catch() instead of async/await and try/catch

Reply:
used then()